process/software-project-lifecycle.md

```diff
@@ -0,0 +1,161 @@
+
+Sandbox projects should be early-stage projects that the OpenSSF TAC or parent Working Group believes warrant experimentation.
+
+* New software focused projects that are designed to extend one or more OpenSSF Projects with functionality or interoperability libraries.
+* Independent software focused projects that fit the OpenSSF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
+* Software focused projects commissioned or sanctioned by the OpenSSF, including initial code for OpenSSF WG collaborations, and "experimental" projects
+* Any software focused project that realistically intends to join OpenSSF Incubation in future and wishes to lay the foundations for that.
+
+#### Project Graduation Process: Sandbox to Incubating
+
+Incubating software focused projects are required to undergo due diligence as a part of the process to move from Sandbox to Incubation. Due Diligence is driven by a TAC or parent WG sponsor, with two weeks for public comment before a vote is called.
+
+#### Submission Forms for Sandbox
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+#### Submission Forms for Sandbox to Incubating
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+### Incubating
+
+![Incubating](/process/incubation-process.png)
+
+#### Gives - Review and Reporting
+*** Need Text ***
+
+#### Gets - Benefits
+*** Need Text ***
+
+#### Incubating Entry Requirements
+
+To be accepted to incubating stage, a project must meet the sandbox stage requirements plus:
+
+* Document that it is being used successfully in production by at least three independent end users which, in the TAC’s judgement, are of adequate quality and scope.
+* Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+* Demonstrate a substantial ongoing flow of commits and merged contributions.
+* Since these metrics can vary significantly depending on the type, scope and size of a project, the TAC has final judgement over the level of activity that is adequate to meet these criteria
+* A clear versioning scheme.
+* Specifications must have at least one public reference implementation.
+
+#### Project Graduation Process: Incubating to Graduation
+
+Software focused projects that wish to move from Incubating to Graduation should open a PR confirming the following criteria:
+* Have committers from at least two organizations.
+* Have achieved and maintained a [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/).
+* Have completed an independent and third party security audit with results published of similar scope and quality as [this example](https://github.com/envoyproxy/envoy#security-audit) which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
```

Comment:
as on #95, I suggest adding words here to indicate that the TAC and/or other WGs may provide assistance with the sec audit funding, which was not true in CNCF when this was written.

Reply:
+1

Reply:
Done.